releasenotes/README.md

```diff
@@ -26,12 +26,28 @@ Additional documentation:
 * [Salesforce CLI Setup Guide](https://developer.salesforce.com/docs/atlas.en-us.sfdx_setup.meta/sfdx_setup/sfdx_setup_intro.htm)
 
 
-## 2.53.6 (August 7, 2024) [stable-rc]
+## 2.54.6 (August 14, 2024) [stable-rc]
+
+**ANNOUNCEMENT:** If you still use the deprecated `sfdx`-style commands, be sure you read [this pinned GitHub issue](https://github.com/forcedotcom/cli/issues/2974) about the upcoming removal of these commands:  `force:source:*`, `force:org:create`, and `force:org:delete`. 
```

Comment:
I think the `sfdx-style` verbage is confusing/misleading.  Many commands will still work by typing `sfdx` or using colons like `force:org:list` or using  flags like `--skipconnectionstatus` 

It's more about whether you use one of the deprecated commands we're planning to remove.

and the removal also affects `force:mdapi` commands!

Reply:
thanks!  will update